_posts/2022-03-16-cockpit-265.md

```diff
@@ -0,0 +1,77 @@
+
+## Crypto policies support
+
+Cockpit now has basic support for displaying and changing a subset of crypto policies (default, future, legacy). The current configured crypto policy is shown in the "Configuration" card.
+
+![158211328-fd8936be-5127-4e30-9c4b-5ba251c7d60a]({{ site.baseurl }}/images/265-crypto-policies-support.png)
+
+Crypto policy changes can be applied directly or later, when later is selected the health card shows that a reboot is required to fully apply the changed crypto policy.
+
+![158211149-36822bc4-a47e-4501-a01a-a82f7bbf3ec2]({{ site.baseurl }}/images/265-crypto-policies-support-2.png)
+
+## Metrics: Show busiest CPU core
+
+On multi-core systems, the Metrics page now shows an additional usage bar for the CPU core with the highest usage. This points out bottlenecks and performance problems which happen because only a few cores (or even a single one) get fully used, while many others are mostly idle. In that case, the average CPU usage will be low.
+
+![screenshot of show busiest cpu core]({{ site.baseurl }}/images/265-metrics-show-busiest-cpu-core.png)
+
+## Machines: Disk serial number
+
+Disks identifier can be specified during a disk attachment. Its use cases include:
+- User might have licensed functionality inside the VM tied to a specific
+disk serial number (most common with Windows guests)
+- Setting up multipath storage
+- This disk will be identifiable from inside the guest by /dev/disk/by-id.
+
+![TestMachinesDisks-testAddDiskAdditionalOptions-vm-add-disk-modal-additional-options-pixels]({{ site.baseurl }}/images/265-machines-disk-serial-number.png)
+
+
+## Try it out
+
+Cockpit 265 and cockpit-machines 264 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/265)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-ad93e6cdcb)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-ec9612ac99)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/264)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-7e1cbde0f7)
+* [cockpit-machines Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-2cd2b4d57d)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
```

Comment:
We didn't actually release a new Client, and none of the new features affect it -- i.e. we don't need to. Let's just drop the line.